code/modules/reagents/chemistry/holder.dm

```diff
@@ -682,8 +714,9 @@ var/const/INJECT = 5 //injection
 	return trans_data
 
 /datum/reagents/proc/get_reagent(type)
-	var/list/cached_reagents = reagent_list
-	. = locate(type) in cached_reagents
+	if(reagent_list)
+		var/list/cached_reagents = reagent_list
+		. = locate(type) in cached_reagents
```

Comment:
I'd test if this if is even needed, iirc locate(type) in list will treat a null list the same as an empty list.